app/controllers/projects/samples_controller.rb

```diff
@@ -40,7 +46,9 @@ def create
 
       if @sample.persisted?
         flash[:success] = t('.success')
-        redirect_to namespace_project_sample_path(id: @sample.id)
+        respond_to do |format|
```

Comment:
You can revert this change and add a data: {turbo: false} to the form signature in `samples/new.html.erb`. It will look like this:

`<%= form_with(model: @sample, url: namespace_project_samples_path, data: {turbo: false}, method: :post) do |form| %>`

This will properly redirect from the new sample page to the show sample page

Reply:
Fixed in 0fb8fc1c66eb29d81ced91306361fadc7cb1d8b3